src/css/components/_on-screen-presets.scss

```
@@ -0,0 +1,64 @@
+.on-screen-presets { 
+    height: 100%;
+    @include display(flex);
+    @include flex-direction(column);
+}
+
+.on-screen-presets-container {
+    width: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.on-screen-presets .soft-buttons {
+    width: 100%;
+    @include display(flex);
+    @include justify-content(space-around);
+    @include flex-direction(row);
+    @include flex-wrap(nowrap);
+    @include align-items(stretch);
+    flex-shrink: 0;
+    height: 100px;
+}
+
+.on-screen-presets .soft-button {
+    width: 100%;
+    border-radius: 0px;
+    margin-bottom: 0px;
+    white-space: nowrap;
+    overflow: hidden;
+    display: inline-block;
+    text-overflow: ellipsis;
+    line-height: 100px;
+    padding: 0px 10px;
+}
+
+.on-screen-presets .progress-bar {
+    position: relative;
+    height: calc((#{$master-height} - 75px) / 12) ;
+
+    border-bottom: 2px solid;
+
+    &__progress {
+        position: absolute;
+        height: 100%;
+        background: color($secondary-accent-color);
+    }
+}
```

```suggestion
.on-screen-presets .progress-bar {
    height: calc((#{$master-height} - 75px) / 12) ;
}
```
Seems that some of the progress bar border issues came from attempting to copy these properties. We should only overwrite the ones we specifically need and leave the rest alone